spec/requests/user/user_settings_spec.rb

```diff
@@ -139,7 +139,7 @@ def send_request(flag = true)
     it "schedules the job while updating" do
       expect do
         post "/users/update_twitch_username", params: { user: { twitch_username: "anna_lightalloy" } }
-      end.to have_enqueued_job(Streams::TwitchWebhookRegistrationJob).exactly(:once).with(user.id)
+      end.to have_enqueued_job(Streams::TwitchWebhookRegistrationWorker).exactly(:once).with(user.id)
```

Comment:
these tests will fail because `have_enqueued_job` uses ActiveJob and we aren't. You can use either `sidekiq_assert_enqueued_jobs` or `sidekiq_assert_enqueued_with`, see https://github.com/thepracticaldev/dev.to/issues/5305#issuecomment-570264567

Reply:
Haha,  I just noticed this and was about to update the PR. You're fast 👅